include/liblwm2m.h

```diff
@@ -786,7 +786,8 @@ struct _lwm2m_context_
     void *                  monitorUserData;
 #endif
 #ifdef LWM2M_BOOTSTRAP_SERVER_MODE
-    lwm2m_client_t *        clientList;
+    lwm2m_client_t *        clientList;     // TODO: this is not used and should be removed??
+    lwm2m_block_data_t *       blockData;   // list to handle temporary block data.
```

Comment:
I think this might limit the server to only bootstrapping one client at a time. If the clientList isn't used, that might already be the case. Have you looked at that at all? If that is the case we should create an issue to correct it.

Reply:
I'm trying to wrap my head around it... I pushed this PR as a work in progress to get some feedback. 

Also having multiple identical "clientList" breaks the ability to build with both
`-DLWM2M_SERVER_MODE` and `-DLWM2M_BOOTSTRAP_SERVER_MODE` at the same, that I know some use.

Reply:
[210330_PR563_parallel_bootstrap.pcapng.zip](https://github.com/eclipse/wakaama/files/6229492/210330_PR563_parallel_bootstrap.pcapng.zip)
Practical testing show that bootstrapping works for two clients in parallel (attached wireshark capture)
```
./build-bootstrap/bootstrap_server -f examples/bootstrap_server/bootstrap_server.ini -4
./build-client/lwm2mclient -4 -b -n bepa2 -l 56830
./build-client/lwm2mclient -4 -b -n bepa3 -l 56831
```

Reply:
I'm not convinced that the testing would show the problem. The first one sends the GET for the final block before the second one sends the first GET. The block requests aren't really interleaved. The data is also the same, so it might be hard to tell if there was a problem from what the server sees.

Reply:
Eventually we should have blocktransfer support in the bootstrap server, but I'm fine waiting on that until other issues are resolved with the blocktransfer support.

Reply:
I think we can hold off with this until we can do it right. One way might be to look into if we can refactor blocktransfer but that would be a bigger task.

Reply:
Getting block transfers right during bootstrap is a bigger task, since there is AFAIK no mechanism to track client contexts during bootstrap. So disabling block transfers during bootstrap seems sensible to me right now.